ui/apps/platform/src/Containers/Clusters/InitBundles/InitBundleForm.tsx

```diff
@@ -83,7 +83,7 @@ function InitBundleForm(): ReactElement {
             generateClusterInitBundle({ name })
                 .then(({ response }) => {
                     setErrorMessage('');
-                    downloadBundle(installation, response); // TODO try catch?
+                    downloadBundle(installation, name, response); // TODO try catch?
```

Comment:
nit: If you update an function signature, it's a good practice to add new parameters to the end of the list. That way, you can add a default parameter to maintain backwards compatibility.
(Other option--but one that adds complexity-- would be to pass arguments in a single Object, so when adding keys, order doesn't matter. Like React props.)

In this case, since this code is not released yet, and the affected area is very small (one spot), it's not a big deal.

Reply:
oops, you're right. i'll keep that in mind later -- i suppose i was thinking more alphabetically in the spur of the moment 😅